utils/api.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+)
+
+// API Constants
+const (
+	APIUsers = "user"
+)
+
+// UserRequest is format for Request to Server
+type UserRequest struct {
+	SnykToken string `json:"snyk_api_token,omitempty"`
+	UUID      string `json:"user_id,omitempty"`
+}
+
+// UserResponse is format for Response from Server
+type UserResponse struct {
+	UUID string `json:"user_id,omitempty"`
+}
+
+// HTTPRequestType is request type to HTTPRequest Method
+type HTTPRequestType struct {
+	ReqData         UserRequest `json:"reqData,omitempty"`
+	Method          string      `json:"method,omitempty"`
```

Comment:
Method and endpoint need not be fields in this as it not required to be passed into the function, the values are already present here.

Reply:
These two fields are not generic.